Line/android/LINELoginDemo/.gitignore

```diff
@@ -0,0 +1,53 @@
+# Based on: https://github.com/github/gitignore/blob/master/Android.gitignore
+
+# Android
+# Built application files
+*.apk
+*.ap_
+
+# Files for the ART/Dalvik VM
+*.dex
+
+# Java class files
+*.class
+
+# Generated files
+bin/
+gen/
+out/
+
+# Gradle files
+.gradle/
+build/
+
+# Local configuration file (sdk path, etc)
+local.properties
+
+# Proguard folder generated by Eclipse
+proguard/
+
+# Log Files
+*.log
+
+# Android Studio Navigation editor temp files
+.navigation/
+
+# Android Studio captures folder
+captures/
+
+# Intellij
+*.iml
+.idea/
+
+# Keystore files
+*.jks
+
+# External native build folder generated in Android Studio 2.2 and later
+.externalNativeBuild
+
+# LINE SDK
+line-android-sdk*.jar
```

Comment:
As LINE SDK is only provided in ZIP for download to developer who've registered an LINE Business account, I'm worry about licensing issue may arise if I include the SDK in the sample. I've pointed to LINE Developers site to acquire the SDK in README file.